tools/chapel-py/src/method-tables/uast-methods.h

```diff
@@ -571,6 +571,18 @@ CLASS_END(NamedDecl)
 CLASS_BEGIN(EnumElement)
   PLAIN_GETTER(EnumElement, init_expression, "Get the init expression of this EnumElement node",
                Nilable<const chpl::uast::AstNode*>, return node->initExpression())
+  PLAIN_GETTER(EnumElement, assigned_value, "Get the 'param' value assigned to this EnumElement node, either directly or via sequence increment",
+               Nilable<const chpl::types::Param*>,
+               auto parentId = node->id().parentSymbolId(context);
```

Comment:
more of a general question than anything about this PR, but how does `computeNumericValuesOfEnumElements` get mapped down during compilation and codegen? Do we just always create a `EnumType` in the production compiler that has all concrete values?

Reply:
Ok. IMO, dyno should just have the logic and we should remove the backend logic so there is no potential for drift. But thats obviously a bigger picture dyno comment than anything about this PR